### ISSUE TITLE
[Bug]: Error display pictures from upload webdav

### ISSUE THREAD

**Issue by fred-gb** [nextcloud/server#36244]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Hi,
Nextcloud instance on VPS
Synology Cloud Sync with webdav folder on Nextcloud

Pictures can't be open on Nextcloud UI when sync by webdav client on Synology shared folder.

### Steps to reproduce

1. Configure Cloud Sync on Synology to access to Nextcloud Share.
2. When I upload picture on Nextcloud UI, picture can be open and sync works on synology and can be open too on Synology shared folder. 
3. When I put a picture on Synology shared folder, sync works, but picture cannot be open on Nextcloud webUI or iOS client.

### Expected behavior

Put picture on Synology shared folder and can be open after on Nextcloud WebUI and show preview.

### Installation method

Community Docker image

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MySQL

### Is this bug present after an update or on a fresh install?

Fresh Nextcloud Server install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "htaccess.RewriteBase": "\/",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "apps_paths": [
            {
                "path": "\/var\/www\/html\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/www\/html\/custom_apps",
                "url": "\/custom_apps",
                "writable": true
            }
        ],
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "password": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "overwriteprotocol": "https",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "localhost",
            "workspace.domain.tld",
            "127.0.0.1"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "25.0.3.2",
        "overwrite.cli.url": "https:\/\/workspace.domain.tld",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "default_phone_region": "FR",
        "default_language": "fr",
        "default_locale": "fr_FR",
        "skeletondirectory": "",
        "knowledgebaseenabled": "false",
        "maintenance": false,
        "mail_sendmailmode": "smtp",
        "mail_smtpmode": "smtp",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpsecure": "ssl",
        "mail_smtpauth": true,
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "loglevel": 2,
        "theme": ""
    }
}
```

### List of activated Apps

```shell
Enabled:
  - activity: 2.17.0
  - announcementcenter: 6.4.0
  - bookmarks: 12.0.0
  - bruteforcesettings: 2.5.0
  - calendar: 4.2.1
  - circles: 25.0.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - contacts: 5.0.2
  - contactsinteraction: 1.6.0
  - dashboard: 7.5.0
  - dav: 1.24.0
  - deck: 1.8.3
  - event_update_notification: 2.0.0
  - external: 5.0.0
  - federatedfilesharing: 1.15.0
  - federation: 1.15.0
  - files: 1.20.1
  - files_accesscontrol: 1.15.1
  - files_antivirus: 4.0.2
  - files_automatedtagging: 1.15.0
  - files_external: 1.17.0
  - files_fulltextsearch: 25.0.0
  - files_fulltextsearch_tesseract: 25.0.0
  - files_mindmap: 0.0.27
  - files_pdfviewer: 2.6.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - forms: 3.0.3
  - fulltextsearch: 25.0.0
  - geoblocker: 0.5.7
  - groupfolders: 13.1.0
  - guests: 2.3.0
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - mail: 2.2.2
  - maps: 0.2.4
  - nextcloud_announcements: 1.14.0
  - notes: 4.6.0
  - notifications: 2.13.1
  - oauth2: 1.13.0
  - onlyoffice: 7.6.8
  - password_policy: 1.15.0
  - passwords: 2023.1.22
  - photos: 2.0.1
  - polls: 4.0.0
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - ransomware_protection: 1.14.0
  - recommendations: 1.4.0
  - related_resources: 1.0.4
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - sharebymail: 1.15.0
  - spreed: 15.0.2
  - support: 1.8.0
  - survey_client: 1.13.0
  - systemtags: 1.15.0
  - text: 3.6.0
  - theming: 2.0.1
  - twofactor_backupcodes: 1.14.0
  - unsplash: 2.1.1
  - updatenotification: 1.15.0
  - user_status: 1.5.0
  - viewer: 1.9.0
  - weather_status: 1.5.0
  - webhooks: 0.4.2
  - workflow_ocr: 1.25.2
  - workflow_script: 1.10.0
  - workflowengine: 2.7.0
Disabled:
  - admin_audit
  - appointments: 1.14.10
  - apporder: 0.15.0
  - electronicsignatures: 2.0.1
  - encryption
  - firstrunwizard: 2.13.0
  - listman: 20.2.2
  - printer
  - socialsharing_email: 2.5.0
  - suspicious_login
  - tasks: 0.14.5
  - timetracker: 0.0.77
  - twofactor_totp
  - user_ldap
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
[PHP] Erreur: Error: imagecreatefromstring(): Data is not in a recognized format at /var/www/html/lib/private/legacy/OC_Image.php#758 at <<closure>>

 0. <<closure>>
    OC\Log\ErrorHandler::onError(2, "imagecreatefrom ... t", "/var/www/html/l ... p", 758)
 1. /var/www/html/lib/private/legacy/OC_Image.php line 758
    imagecreatefromstring("*** sensitive parameters replaced ***")
 2. /var/www/html/lib/private/Preview/Image.php line 52
    OC_Image->loadFromFile("/var/www/html/d ... g")
 3. /var/www/html/lib/private/Preview/GeneratorHelper.php line 65
    OC\Preview\Image->getThumbnail(OC\Files\Node\File {}, 4096, 4096)
 4. /var/www/html/lib/private/Preview/Generator.php line 343
    OC\Preview\GeneratorHelper->getThumbnail(OC\Preview\JPEG {}, OC\Files\Node\File {}, 4096, 4096)
 5. /var/www/html/lib/private/Preview/Generator.php line 162
    OC\Preview\Generator->getMaxPreview(OC\Files\SimpleFS\SimpleFolder {}, OC\Files\Node\File {}, "image/jpeg", "")
 6. /var/www/html/lib/private/Preview/Generator.php line 114
    OC\Preview\Generator->generatePreviews(OC\Files\Node\File {}, [[250,250,true,"fill"]], "image/jpeg")
 7. /var/www/html/lib/private/PreviewManager.php line 185
    OC\Preview\Generator->getPreview(OC\Files\Node\File {}, 250, 250, true, "fill", null)
 8. /var/www/html/core/Controller/PreviewController.php line 144
    OC\PreviewManager->getPreview(OC\Files\Node\File {}, 250, 250, true, "fill")
 9. /var/www/html/core/Controller/PreviewController.php line 113
    OC\Core\Controller\PreviewController->fetchPreview(OC\Files\Node\File {}, 250, 250, false, true, "fill")
10. /var/www/html/lib/private/AppFramework/Http/Dispatcher.php line 225
    OC\Core\Controller\PreviewController->getPreviewByFileId(30557, 250, 250, false, true, "fill")
11. /var/www/html/lib/private/AppFramework/Http/Dispatcher.php line 133
    OC\AppFramework\Http\Dispatcher->executeController(OC\Core\Controller\PreviewController {}, "getPreviewByFileId")
12. /var/www/html/lib/private/AppFramework/App.php line 172
    OC\AppFramework\Http\Dispatcher->dispatch(OC\Core\Controller\PreviewController {}, "getPreviewByFileId")
13. /var/www/html/lib/private/Route/Router.php line 298
    OC\AppFramework\App::main("OC\\Core\\Controller\\PreviewController", "getPreviewByFileId", OC\AppFramework\ ... {}, ["core.Preview.getPreviewByFileId"])
14. /var/www/html/lib/base.php line 1047
    OC\Route\Router->match("/core/preview")
15. /var/www/html/index.php line 36
    OC::handleRequest()

GET /core/preview?fileId=30557&x=250&y=250
from 37.170.125.190 by franchini at 2023-01-19T14:32:17+00:00
```

### Additional info

Maybe it's duplicate issue. But I don't find solution.
When I try to mount with external storage, SFTP, it's very very slow.

Thanks

**Comment by fred-gb** [comment]
I also tried:
From Nextcloud mount external storage with WebDAV from NAS:
```
[webdav] Erreur: Sabre\DAV\Exception\ServiceUnavailable: Storage with mount id 9 is not available at <<closure>>

0. /var/www/html/3rdparty/sabre/dav/lib/DAV/Tree.php line 78
   OCA\DAV\Connector\Sabre\Directory->getChild("chantiers")
1. /var/www/html/3rdparty/sabre/dav/lib/DAV/Tree.php line 73
   Sabre\DAV\Tree->getNodeForPath("files/folder1/folder3")
2. /var/www/html/apps/dav/lib/Connector/Sabre/LockPlugin.php line 68
   Sabre\DAV\Tree->getNodeForPath("files/franchini ... g")
3. /var/www/html/3rdparty/sabre/event/lib/WildcardEmitterTrait.php line 89
   OCA\DAV\Connector\Sabre\LockPlugin->getLock(Sabre\HTTP\Request {}, Sabre\HTTP\Response {})
4. /var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php line 456
   Sabre\DAV\Server->emit("beforeMethod:PUT", [Sabre\HTTP\Requ ... }])
5. /var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php line 253
   Sabre\DAV\Server->invokeMethod(Sabre\HTTP\Request {}, Sabre\HTTP\Response {})
6. /var/www/html/3rdparty/sabre/dav/lib/DAV/Server.php line 321
   Sabre\DAV\Server->start()
7. /var/www/html/apps/dav/lib/Server.php line 360
   Sabre\DAV\Server->exec()
8. /var/www/html/apps/dav/appinfo/v2/remote.php line 35
   OCA\DAV\Server->exec()
9. /var/www/html/remote.php line 171
   require_once("/var/www/html/a ... p")

PUT /remote.php/dav/files/folder1/folder3/Manger5675.jpg
from 37.166.84.11 by franchini at 2023-01-19T20:19:21+00:00
```

On iOS client, I have 503 in loop. When I tried to upload.

And with external stoge in SFTP mode, not working too. iOS client can't upload without 500 error.

I can't use in production.

Thanks

**Comment by kesselb** [comment]
Fixed by #39093
